src/annotator/util/frame-util.js

```diff
@@ -43,6 +43,16 @@ function isValid (iframe) {
   return iframe.className !== 'h-sidebar-iframe';
 }
 
+function isReady (iframe, callback) {
```

Comment:
There's something related to this that I haven't explored yet since we are focused on an MVP, but I would like to see if we can simplify things to just rely on DOMContentLoaded. It might not be critical that we wait for a full load for the majority of the cases, but I don't know for sure.